### PR TITLE
set hostpath version

### LINKED ISSUE #191: [Enhancement]: hostpath image should not be latest
Set the hostpath image to a version rather than latest

#### Design Document Link

PR: TBD

#### What would you like to be added:
Set version of hostpath and verify it works

#### Why is this needed:
Using latest may set us up for issues in the future

### PR DESCRIPTION
**Pull request by cooktheryan**
**Which issue(s) this PR addresses**:

Closes #191

### CODE REVIEW

File: pkg/assets/apps/bindata.go [hunk 1]
```diff
@@ -863,7 +863,7 @@ spec:
       serviceAccountName: kubevirt-hostpath-provisioner-admin
       containers:
         - name: kubevirt-hostpath-provisioner
-          image: quay.io/kubevirt/hostpath-provisioner
+          image: quay.io/kubevirt/hostpath-provisioner:v0.8.0
```
Comment (rootfs) [inline pkg/assets/apps/bindata.go:866]: don't edit this file. It is generated from https://github.com/redhat-et/microshift/blob/main/assets/apps/000_80_hostpath-provisioner-daemonset.yaml by https://github.com/redhat-et/microshift/blob/main/scripts/bindata.sh

**Comment by rootfs** [comment]
cool, please run a pv test and see if the image works https://github.com/redhat-et/microshift/blob/main/test/pv-test.sh

**Comment by cooktheryan** [comment]
```
Error from server (NotFound): namespaces "test-pv" not found
namespace/test-pv created
persistentvolume/hostpath-provisioner created
persistentvolumeclaim/kube-1-hpp-claim created
pod/test-pod created
pod/test-pod condition met
test passed
[fedora@ip-172-31-5-65 microshift]$ kubectl get pvc -n test-pv
NAME               STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                    AGE
kube-1-hpp-claim   Bound    pvc-ee592b37-1e0c-4b28-8759-81e3d53e46c2   98Gi       RWO            kubevirt-hostpath-provisioner   25s
[fedora@ip-172-31-5-65 microshift]$ kubectl get po kubevirt-hostpath-provisioner-p9drd -n kubevirt-hostpath-provisioner -o yaml | grep image
    image: quay.io/kubevirt/hostpath-provisioner:v0.8.0
    imagePullPolicy: Always
  imagePullSecrets:
    image: quay.io/kubevirt/hostpath-provisioner:v0.8.0
    imageID: quay.io/kubevirt/hostpath-provisioner@sha256:8cf5c2186a0357a70b835f2ef76430306e91da5743f375f87693f61a470ac75f

**Pull request merged**